src/main/java/org/eln2/mc/utility/McColor.kt

```diff
@@ -0,0 +1,67 @@
+package org.eln2.mc.utility
+
+import net.minecraft.network.FriendlyByteBuf
+
+class McColor(val r : UByte, val g : UByte, val b : UByte, val a : UByte) {
+    constructor(r : Int, g : Int, b : Int, a : Int) : this(
+        if (r < 256 && r > -1) r.toUByte() else throw Exception("R is not within bounds."),
```

Comment:
Grissess mentioned in one of my PRs that Codacy likes the pattern `error(<msg>)` rather than `throw Exception(<msg>)`. Perhaps we can swap these over too?

Reply:
I've added #184 to the issue tracker to replace all of these instances, since I'm sure there are plenty of them.